### PR TITLE
CU-86drw584r Swap Multi Invoke - Implement SwapController without routes

### PR DESCRIPTION
**Pull request by lopescode**

### CODE REVIEW

**Comment by melanke** [comment]
Task linked: [CU-86drw584r Swap Multi Invoke - Implement SwapController without routes](https://app.clickup.com/t/86drw584r)

File: packages/bs-neo3/src/FlamingoSwapControllerService.ts [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+      throw new FlamingoInvalidReservesResponseError()
+
+    this.reservesToReceive = stack[0].value[0].value
+    this.reservesToUse = stack[0].value[1].value
+
+    this.recalculateSwapArguments()
+  }
+
+  startListeningBlockGeneration() {
+    this.ws = new WebSocket('wss://rpc10.n3.nspcc.ru:10331/ws')
```
Comment (raulduartep) [inline packages/bs-neo3/src/FlamingoSwapControllerService.ts:185]: Create a new constant for this url

File: packages/bs-neo3/src/FlamingoSwapHelper.ts [hunk 2]
```diff
@@ -0,0 +1,171 @@
+import BigNumber from 'bignumber.js'
+import { Network, SwapRoute, Token } from '@cityofzion/blockchain-service'
+import { SWAP_SCRIPT_HASHES_BY_NETWORK_TYPE, TOKENS } from './constants'
+
+export type SwapAccountBalance = {
+  token: Token
+  amount: string
+  amountNumber: number
+}
```
Comment (raulduartep) [inline packages/bs-neo3/src/FlamingoSwapHelper.ts:9]: Are you using it ?

File: packages/bs-neo3/src/FlamingoSwapNeonDappKitInvocationBuilder.ts [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+        {
+          type: 'Integer',
+          value: deadline,
+        },
+      ],
+    })
+
+    const isNeoSwapped = tokenToReceive.hash === scriptHashes.neo
+    if (isNeoSwapped) {
+      const GAS_PER_NEO = 0.001
```
Comment (raulduartep) [inline packages/bs-neo3/src/FlamingoSwapNeonDappKitInvocationBuilder.ts:122]: Insert it into the constants folder

File: packages/bs-neo3/src/FlamingoSwapControllerService.ts
Comment (raulduartep) [inline packages/bs-neo3/src/FlamingoSwapControllerService.ts:1]: What do you think about creating a new folder to put all these swap files?

**Pull request merged**